src/tic_tac_toe/player.clj

```diff
@@ -1,6 +1,18 @@
 (ns tic-tac-toe.player
   (:use [tic-tac-toe.user-interface :as ui]))
 
-(defn pick-move
+(defn move-input
   []
   (- (ui/parse-int (ui/get-input)) 1))
+
+(defn pick-next-open-spot
+  [board]
+  (loop [i 0
+         board board]
+  (if (= (board i) " ")
+    i
+    (recur (inc i) board))))
+
+(defmulti pick-move (fn [type & args] (:type type)))
```

Comment:
I realize this currently passes an additional argument to the human player which it doesn't yet use.  I plan to incorporate the board for future input validation, so I went with this choice to take advantage of the runtime polymorphism.

Reply:
🤔 I feel like that having the human player perform the validation is just finding a way to accommodate the interface of the computer move